src/amp.js

```diff
@@ -71,6 +71,9 @@ const shouldMainBootstrapRun = !self.IS_AMP_ALT;
  * @param {!./service/performance-impl.Performance} perf
  */
 function bootstrap(ampdoc, perf) {
+  delete window['IntersectionObserver'];
```

Comment:
@dvoytenko I added this to test in Chrome. Got the same result if I disable inob in firefox.